lib/components/viewers/trip-viewer.js

```diff
@@ -1,29 +1,30 @@
 // FIXME: Remove this eslint rule exception.
 /* eslint-disable jsx-a11y/label-has-for */
+import { Button, Label } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import { FormattedMessage, FormattedTime } from 'react-intl'
 import coreUtils from '@opentripplanner/core-utils'
 import moment from 'moment'
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { Button, Label } from 'react-bootstrap'
-import { FormattedMessage, FormattedTime } from 'react-intl'
-import { connect } from 'react-redux'
 
+import * as mapActions from '../../actions/map'
+import * as uiActions from '../../actions/ui'
+import { findGraphQLTrip, findTrip } from '../../actions/api'
```

Comment:
Do the same for API actions.

Reply:
Slept right through that one! Thank you for catching

Extracted in 24d33b2cc7bc6c907163261e887bc141de279f1d